src/any_agent/serving/agent_card.py

```diff
@@ -0,0 +1,37 @@
+import inspect
+from typing import TYPE_CHECKING
+
+from common.types import AgentCapabilities, AgentCard, AgentSkill
+
+from any_agent.config import MCPParams
+
+if TYPE_CHECKING:
+    from any_agent import AnyAgent
+    from any_agent.config import ServingConfig
+
+
+def _get_agent_card(agent: "AnyAgent", serving_config: "ServingConfig") -> AgentCard:
+    skills = []
+    for tool in agent.config.tools:
+        # TODO: handle MCP tools
+        if isinstance(tool, MCPParams):
+            continue
+        skills.append(
+            # TODO: find what other arguments can be set.
+            AgentSkill(
+                id=f"{agent.config.name}-{tool.__name__}",
+                name=tool.__name__,
+                description=inspect.getdoc(tool),
+            )
+        )
+    return AgentCard(
+        name=agent.config.name,
+        description=agent.config.description,
+        version=serving_config.version,
+        url=f"http://{serving_config.host}:{serving_config.port}/",
+        # TODO: extend default capabilities
```

Comment:
I wanted to keep this to the minimum for now. 

Adding `streaming` capabilities would require us to implement some sort of `run_stream` method in AnyAgent, which I am not sure it's worth it.

I haven't check what the other 2 capabilities mean in practice.